platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/events/performance/JankStatsMonitor.kt

```diff
@@ -157,7 +159,17 @@ internal class JankStatsMonitor(
             JankFrameType.ANR
         }
 
-    private fun FrameData.durationToMilli(): Long = this.frameDurationUiNanos / TO_MILLI
+    private fun FrameData.durationToMilli(): Long {
```

Comment:
also an un-related reminder to move the `runtime.isEnabled(RuntimeFeature.DROPPED_EVENTS_MONITORING)` check that happens `onFrame()` inside the `if (volatileFrameData.isJank) {}` check